### PR TITLE
chore: Privacy Policy

### PR DESCRIPTION
**Pull request by BlueAcidFrog**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: src/markdown/pages/privacy.md [hunk 1]
```diff
@@ -1,96 +1,214 @@
...
+Cookies can contain the following information about you and your use of the Website: browser type, search preferences, data relating to which pages of the Website that you have visited and the date and time of your use. Ory uses Cookies for the following purposes: 
+
+ - To enable and support security features, prevent fraud, and protect your data from unauthorized access.
+ - To enable features and help us provide you with personalized content.
+ - To analyze how you use the Website and to monitor site performance. These Cookies help us to identify and fix errors, understand and improve services, research and test out different features, and monitor how you reached the Website.
+
+### Third-Party Cookies 
+Ory uses the following third-party Cookies on the Website:  
+
+Tabelle Fehlt
```
Comment (BlueAcidFrog) [inline src/markdown/pages/privacy.md:121]:
```suggestion
| Cookie Name          | Type        | Expiry   | Use                                                                                                                                                                                                                                                                                                                                         |
| -------------------- | ----------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Google Analytics     | Required    | 2 years  | Analytics services designed keep track of page traffic and user behavior while browsing the Website. We use this data internally to improve and enhance the usability and performance of the Website. Disabling this Cookie makes it more difficult for us to understand how our Website is being used and improve and enhance the Website. |
| Hubspot              | Required    | 13 month | HubSpot develops cloud-based, inbound marketing software that allows businesses to transform the way that they market online                                                                                                                                                                                                                |
| Google Tag Manager   | Functional  | N/A      | Google Tag Manager allows us to determine how effective our advertising campaigns have been by measuring how many users arrived at our Website via external advertisements for our Services.                                                                                                                                                |
| LinkedIn Insight Tag | Advertising | 30 days  | This Cookie allows us to determine how effective our LinkedIn campaigns are by measuring how many users have clicked prompts on LinkedIn to arrive at our Website.                                                                                                                                                                          |
| Twitter Insight Tag  | Advertising | 30 days  | This Cookie allows us to determine how effective our Twitter campaigns are by measuring how many users have clicked prompts on Twitter to arrive at our Website.                                                                                                                                                                            |
```

**Review by aeneasr — APPROVED** [review]
👌

**Comment by aeneasr** [comment]
@BlueAcidFrog from my side this is good to go! Merge?

**Comment by jfcurran** [comment]
@BlueAcidFrog ci was failing because the pr didnt have a [semantic](https://github.com/zeke/semantic-pull-requests) title. the allowed types are:
  - feat
  - fix
  - docs
  - style
  - refactor
  - perf
  - test
  - build
  - ci
  - chore
  - revert

**Comment by jfcurran** [comment]
r2g 👍

**Comment by jfcurran** [comment]
Screenshot 2021-12-06 at 11 00 24
table seems to be a mishap
Screenshot 2021-12-06 at 11 00 52
also, semicolons in titles seem off - maybe @vinckr can tend to this

**Pull request merged**